analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/ComplianceAnalyzerPlugin.java

```diff
@@ -0,0 +1,278 @@
+package eu.fasten.analyzer.complianceanalyzer;
+
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1ConfigMap;
+import io.kubernetes.client.openapi.models.V1Service;
+import org.pf4j.PluginWrapper;
+import org.pf4j.Plugin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.pf4j.Extension;
+
+import eu.fasten.core.plugins.DBConnector;
+import eu.fasten.core.plugins.KafkaPlugin;
+import eu.fasten.core.data.graphdb.RocksDao;
+
+import java.io.*;
+import java.util.List;
+import java.util.Optional;
+import java.util.Collections;
+import org.jooq.DSLContext;
+import org.json.JSONObject;
+
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.util.KubeConfig;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.openapi.apis.BatchV1Api;
+import io.kubernetes.client.openapi.models.V1Job;
+import io.kubernetes.client.util.Yaml;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.common.collect.Lists;
+
+import io.kubernetes.client.util.Config;
+
+
+/**
+ *   Plugin which runs qmstr command line tool to detect
+ *   license compatibility and compliance.
+ */
+public class ComplianceAnalyzerPlugin extends Plugin {
+
+    /**
+     * Kubernetes namespace where our demo objects are going to be deployed in.
+     */
+    protected static final String K8S_NAMESPACE = "default";
+
+    /**
+     * How will the patched Job file be called.
+     */
+    private static final String PATCHED_JOB_FILE_NAME = "patchedJob.yaml";
+
+    public ComplianceAnalyzerPlugin(PluginWrapper wrapper) {
+        super(wrapper);
+    }
+
+    @Extension
+    public static class CompliancePluginExtension implements KafkaPlugin, DBConnector {
```

Comment:
I agree with @ilyagrishkov. Even though we have a plan to write data to the database in the future, I understood that we should use Kafka for it.

In any case, since the partners didn't agree with a standard way to do so, I'd just remove the `DBConnector`.

Reply:
Removed in https://github.com/fasten-project/fasten/commit/7975246450da52793713310dc6123c14c876daf5.